src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private static final int BOUND = 10;
+    private static final int RADIUS = 10; // Default Circle radius
+    private static final String COLOR = "WHITE"; // Default Circle color
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(5);
```

Comment:
not fixed. Please also introduce a contant for this magic number

Reply:
I don't understand what I have to fix. if I put 4 instead of 5, case 4 will not work